data/mods/Aftershock/mobs/monster_faction.json

```diff
@@ -21,5 +21,16 @@
     "neutral": "herbivore",
     "by_mood": [ "insect", "small_animal", "player" ],
     "hate": "zombie"
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "moxie",
+    "neutral": "bio_machine",
+    "hate": [ "small_animal", "herbivore", "insect", "player" ]
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "bio_machine",
+    "neutral": [ "moxie", "small_animal", "herbivore", "insect", "player" ]
```

Comment:
Do we want all bio_machines ignoring the player unless the player attacks it?

Reply:
The idea is that bio-machines would be the docile form of moxphores so they can add flavor and populate the world, and will only either attack or flee if attacked themselves, I didn't want all bio-machines to be inherently hostile to the player.